workloads/api_load/common.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+source env.sh
+source ../../utils/common.sh
+source ../../utils/benchmark-operator.sh
+
+openshift_login
```

Comment:
Do we need to login to openshift for this? I don't believe we are running this against an OCP cluster right?